pkg/network/netns/netns.go

```diff
@@ -0,0 +1,45 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package netns
+
+import (
+	"fmt"
+
+	"github.com/containernetworking/plugins/pkg/ns"
+)
+
+type NetNS struct {
+	nspath string
+}
+
+func New(pid int) NetNS {
+	return NetNS{nspath: fmt.Sprintf("/proc/%d/ns/net", pid)}
```

Comment:
Since this code is common to here and to the isolation. Maybe it worth moving it to some common place instead of copy pasting it.

Reply:
Which code is in common? I extracted the network stuff from the isolation package.

Reply:
isolation still has `NetNamespace` [method](https://github.com/kubevirt/kubevirt/pull/6921/files#diff-213590bde8eb8a232be5402e83a3f7a393692f51c2c1253330a0adc6f51c29c4R125).

Reply:
Nice catch.
This is a leftover, I removed it now. Let me know if it is ok now.